src/preset.js

```diff
@@ -8,25 +8,105 @@
 genotet.preset = {};
 
 /**
+ * Parameters for preset expression view.
+ * @private @const {!genotet.ExpressionViewParams}
+ */
+genotet.preset.EXPRESSION_PARAMS_ = {
+  fileName: 'expressionMatrix',
+  isGeneRegex: true,
+  isConditionRegex: true,
+  geneInput: 'sig.*',
+  conditionInput: 'si.*',
+  isPreset: true
```

Comment:
Link manager can store this information elsewhere in its own book-keeping. Views should be unaware of anything related to linked queries.